ds3-cli-certification/src/main/java/com/spectralogic/ds3cli/certification/CertificationWriter.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * ******************************************************************************
+ *   Copyright 2014-2016 Spectra Logic Corporation. All Rights Reserved.
+ *   Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *   this file except in compliance with the License. A copy of the License is located at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file.
+ *   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *   specific language governing permissions and limitations under the License.
+ * ****************************************************************************
+ */
+
+package com.spectralogic.ds3cli.certification;
+
+import ch.qos.logback.classic.Logger;
+import com.spectralogic.ds3cli.Arguments;
+import com.spectralogic.ds3cli.CommandResponse;
+import com.spectralogic.ds3client.Ds3Client;
+import org.slf4j.LoggerFactory;
+
+import java.io.*;
+import java.util.Date;
+
+import static com.spectralogic.ds3cli.helpers.Util.command;
+
+
+public class CertificationWriter {
+
+    private static final String EXT = ".html";
+    private static final Logger LOG =  (Logger) LoggerFactory.getLogger(CertificationWriter.class);
+
+    private final OutputStreamWriter OUT;
+    private long currentTimeTag = 0L;
+
+    public CertificationWriter(final String title) throws IOException {
+        OUT = new OutputStreamWriter(new FileOutputStream(title + EXT), "UTF-8");
+        writeHeader(title);
+    }
+
+    public  void writeHeader(final String title) throws IOException {
+        LOG.info("NEW FILE: {}", title);
+        OUT.write(String.format("<html><head><title>%s</title></head>\n", title));
+        OUT.write(String.format("<body class=\"testbody\"><h1 class=\"maintitle\">%s</h1>\n", title));
+    }
+
+    public  void startNewTest(final String testTitle ) throws IOException {
+        currentTimeTag = new Date().getTime();
```

Comment:
Denver set his up to run in order. I did not in Section 8. The "currentTimeTag" is reset in startNewTest() so it that subsequent logs in the current test print a relative span.